### DISCUSSION TITLE
Import specific folder via the photoprism import CLI command

### DISCUSSION THREAD

**Discussion by cluelesscactus** [photoprism/photoprism#1606] [Q&A] [answered]
Hello!

I was curious if it's possible to import specific folders instead of what seems like the base folder that's configured for the imports folder.  I know you can do it via the UI by drilling down and selecting the folders to import but I was curious if this was also possible via the CLI command [referenced here from the Pure Docker Guide](https://docs.photoprism.org/getting-started/docker/#command-reference)

When I have ran it previously it just picks up all content that is under the /photoprism/import folder that is mounted via a docker volume from my NAS but I would like to be able to "surgically" be able to select the specific folder I want to have imported that is under that directory tree on my NAS.  (example.. a folder that is at /photoprism/import/device_name/date instead of just all of /photoprism/import, something like `docker exec -ti photoprism photoprism import /photoprism/import/device_name/date`).  Thanks and loving it so far!

**Comment by lastzero** [comment, marked as answer]
The photoprism cp & mv commands can import from all folders, however containers are specifically designed to put the app in a jail and separate it from your host. So you can't just access any file, you'd have to mount it. It may be easier to copy files to a statically mounted import folder therefore unless you wait for a version that can be installed directly on your host

**Reply by cluelesscactus** [reply]
Yeah, sorry for not fully explaining my setup.

I currently have a few mounts setup in my docker compose to mount a folder for originals and import. 

```
- "/mnt/shares/freenas-share/photoprism/originals:/photoprism/originals"
- "/mnt/shares/freenas-share/photo_library:/photoprism/import"
```

I have a whole other setup/sync/upload process with my devices and NAS and it will end up putting new photos in `/mnt/shares/freenas-share/photo_library/$device_name/$date` folders every day.

So my current Photoprism setup can see the new files just fine, but now I'm looking to add a cron/scheduled job to import just that single new folder every day after it's uploaded.  I didn't know if it was possible to run a command that would ONLY import a specific folder (that I would control via vars in my scheduled job)

Right now it seems to import everything which takes my server a few hours just saying.. duplicate, duplicate, duplicate.. (close to 400gb currently)

I hope this makes a little sense on why I was looking to run the photoprism import command with a specific folder.

**Reply by lastzero** [reply]
An absolute path to import as first command argument should work.
